### PR TITLE
Trace log compressor docs

### PR DESCRIPTION
**Pull request by TTianshun**
I've tried to write the basic usage docs for `VCompressor`. My English is poor. If the docs are not good,  you can close this pr.

### CODE REVIEW

File: docs/source/basic_usage.rst [hunk 1]
```diff
@@ -254,3 +254,20 @@ interact with it. Even better, you can **go back in time** because you know what
     vdb <your_json_report>
 
 For detailed commands, please refer to :doc:`virtual_debug`
+
+Compress Your Report
+--------------------
+
+VizTracer supports compressing your json report. The compress rate is better than commonly used compression methods like ``zlib`` and ``lzma``.
```
Comment (gaogaotiantian) [inline docs/source/basic_usage.rst:261]: The documentation does not need to say it's better than the common methods. However, we can give a general compress rate here.

**Review by gaogaotiantian — CHANGES_REQUESTED** [review]
Just one comment and it should be fine.

File: docs/source/basic_usage.rst (hunk 1 above)
Comment (TTianshun) [reply docs/source/basic_usage.rst]: updated here!

File: docs/source/basic_usage.rst [hunk 2]
```diff
@@ -254,3 +254,20 @@ interact with it. Even better, you can **go back in time** because you know what
     vdb <your_json_report>
 
 For detailed commands, please refer to :doc:`virtual_debug`
+
+Compress Your Report
+--------------------
+
+VizTracer supports compressing your json report. The general compress ratio is about 1% - 3% for a large report.
```
Comment (gaogaotiantian) [inline docs/source/basic_usage.rst:261]: I think the compression ratio is defined as `uncompressed size / compressed size` [wiki](https://en.wikipedia.org/wiki/Data_compression_ratio). Here we could say 50:1 to 100:1 maybe.

File: docs/source/basic_usage.rst [hunk 3]
```diff
@@ -254,3 +254,20 @@ interact with it. Even better, you can **go back in time** because you know what
...
+
+You can compress your report with ``--compress``.
+
+.. code-block:: 
+
+    viztracer -o result.cvf --compress result.json
```
Comment (gaogaotiantian) [inline docs/source/basic_usage.rst:267]: I think maybe ``viztracer --compress result.json -o result.cvf`` makes more sense to the users. Compress then set the output file, compared to set the output file then compress.

File: docs/source/basic_usage.rst [hunk 4]
```diff
@@ -254,3 +254,20 @@ interact with it. Even better, you can **go back in time** because you know what
...
+
+You can also decompress your report with ``--decompress``
+
+.. code-block:: 
+
+    viztracer -o result.json --decompress result.cvf
```
Comment (gaogaotiantian) [inline docs/source/basic_usage.rst:273]: Same thing here for the order.

**Review by gaogaotiantian — CHANGES_REQUESTED** [review]
Compression ratio and one tiny order thing I noticed

**Comment by TTianshun** [comment]
> Compression ratio and one tiny order thing I noticed

Those advices sound reasonable. I've changed that.

**Pull request merged**